node/packages/parser/src/ethereum/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const SHAPE_SHIFT_ROUTER_CONTRACT = '0x6a1e73f12018D8e5f966ce794aa2921941feB17E'
+export const YEARN_V2_ROUTER_CONTRACT = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
+export const YEARN_LINK_TOKEN_VAULT = '0x671a912C10bba0CFA74Cfc2d6Fba9BA1ed9530B2'
```

Comment:
Do we need the specific LINK Vault in constants? Makes it seem like its a special vault.